components/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { ReactComponent as Checked } from '../assets/images/checked.svg'
+
+type Props = {
+  children?: React.ReactNode
+  success?: boolean
+  text?: string
+  title: string
+}
+
+const Card: React.FC<Props> = props => (
+  <div className="row mt-5">
+    <div className="col-sm-5 m-auto" style={{ maxWidth: '604px' }}>
```

Comment:
```
const colClass = `col-sm-${props.col || 6}`
const offsetClass = `col-offset-${props.offset || 0}`
```

Reply:
Offset do not work with margin- auto (`m-auto`) and do not center properly in the middle.
I improved the responsive of the **Card** and instead of just changing `col` and `offset` user can use his own Boostrap classes.